### ISSUE TITLE
[Question] Lambda value for velocity in Loss fn

### ISSUE THREAD

**Issue by vjg28** [carla-simulator/imitation-learning#79]
I have some questions regarding the inclusion of velocity in the loss function. The model has a separate branch that gives predicted velocity value as output give the input from the perception module. 

How is the velocity value included in the loss function? Is it just one extra variable in the mean squared loss or something else. 
```
loss = lambda_velocity * || v- v_gt ||² + lambda_steering * || s - s_gt ||² + lambda_acc * || a - a_gt ||² + lambda_brake * || b - b_gt ||²
```
If it is so, then what is the value of **lambda_velocity** used ? It wasn't mentioned in the previous issues regarding lambda values. @felipecode any sort of clarification would help.

And isn't the speed prediction regularization a feature of your other paper `Exploring the Limitations of Behavior Cloning for Autonomous Driving` to tackle inertia problem or was it also mentioned in CORL paper?
